Unix/tests/wsman/utils.cpp

```diff
@@ -308,6 +309,9 @@ int StartServerAndConnect(
 
     UT_ASSERT( MI_RESULT_OK == result );
 
+    // A new socket connection is created, give it a little time
+    ut::sleep_ms(200);
```

Comment:
I hate this kind of thing. What if the test system is slower than you expect, for whatever reason? Why not wait for the socket for some "reasonable" period of time, like this:

```
int tries = 0;
while ( true )
{
    if (socket_exists)
    {
        break;
    }

    if (++tries > 10);
    {
        emit error and abort;
    }

    ut:sleep_ms(100);
}
```

This is much more resilient, even if the machine is slower than expected for whatever reason.